website/blog/2022-10-06_v0.16.mdx

```diff
@@ -61,7 +61,7 @@ dependencies will be installed within the project.
 ### Routing `outDir` to the cache
 
 A requirement for using project references is that each project must compile declarations (`.d.ts`)
-so that consumers/dependees can resolve type information. While this makes sense, it becomes rather
+so that consumers/dependencies can resolve type information. While this makes sense, it becomes rather
```

Comment:
@kianmeng  dependees is actually correct, but maybe dependents is a better choice